src/Xablu.WebApiClient/Attributes/QueryName.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+namespace Xablu.WebApiClient.Attributes
+{
+    [AttributeUsage(AttributeTargets.All, AllowMultiple = true)]
```

Comment:
I'm not sure `AllowMultiple = true` applies in this case, can you have more than one alias / "name in query" for a single field?

Reply:
fixed